elog/pswww.py

```diff
@@ -109,7 +111,7 @@ def get_facilities_logbook(self, instrument):
             web.get_facilities_logbook('CXI Instrument')
         """
         # Format correct URL
-        url = (self._url + '/lgbk/lgbk/' + instrument + '/ws/info')
+        url = (self._url + f'/{self._lgbk}/lgbk/' + instrument + '/ws/info')
```

Comment:
Might be worthwhile to have `f"{self._url}/{self._lgbk}"` available as a property (`logbook_base_url` or something that overlaps less with the current verbiage?) and reuse it elsewhere